templates/create.jinja2

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
```

Comment:
you should be using the jinja `extends` directive here to allow yourself to skip writing like 90% of this template.